tools/iothub-explorer/package.json

```diff
@@ -15,7 +15,8 @@
     "colors-tmpl": "^1.0.0",
     "nopt": "^3.0.4",
     "prettyjson": "^1.1.3",
-    "uuid": "^2.0.1"
+    "uuid": "^2.0.1",
+    "util": "0.10.3"
```

Comment:
not too sure if this needs to be added here, I thought util was a native node module.